vmdb/spec/models/vm_redhat_spec.rb

```diff
@@ -56,4 +56,18 @@
       expect(vm_redhat.cloneable?).to eq(true)
     end
   end
+
+  context "#supports_migrate?" do
+    it "returns false for RedHat VM" do
+      vm =  FactoryGirl.create(:vm_redhat)
+      expect(VmRedhat.supports_migrate?(vm.id)).to eq(false)
```

Comment:
Let's only test the class method in VmOrTemplate spec since that's the only place that defines this method.

Please change this to be like the other spec below and test the instance method.
`expect(vm. supports_migrate?).to eq(false)`